pkgs/development/libraries/libnitrokey/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, hidapi
+, libusb1
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
why?

Reply:
No specific reason besides that this is more common.

Reply:
Function-style derivation were added after `rec` in an objective to overcome `rec` limitations, I would argue to keep this change and not apply it.